docs/config/development/feature-from-scratch/index.rst

```diff
@@ -0,0 +1,385 @@
+        :emphasize-lines: 5-10
+
+        id: YOUR_PACK_ID
+
+        ...
+
+        stages:
+          - id: trees
+            type: FEATURE
+
+          - id: landforms
+            type: FEATURE
+
+    We'll only be using the ``trees`` stage for this guide.
+
+    .. tip::
+
+        The feature stage ids can be named to your liking and feature stages will generate in order from top to bottom.
+
+2. Create your structure file
+-----------------------------
+
+.. card::
+
+    Structure files can either be dynamic TerraScript ``.tesf`` or static schematics ``.schem`` files.
+
+    .. tab-set::
+
+            .. tab-item:: TerraScript
+
+                TerraScript files consist of :doc:`TerraScript </config/documentation/terrascript/index>` language
+                that typically generate structures procedurally and can create unique complex structure layouts.
+
+                1. Add the ``structure-terrascript-loader`` addon to the pack manifest, using versions ``1.+``
+
+                2. Create a blank ``.tesf`` file.
+
+                3. Add TerraScript within the .tesf file to generate the structure.
+
+                ``oak_tree.tesf`` will be used for this guide.
+
+            .. tab-item:: Schematic
+
+                Schematic files consist of an arrangement of blocks that make up a structure that can be saved through
+                `WorldEdit <https://worldedit.enginehub.org/en/latest/usage/clipboard/>`__.
+
+                1. Add the ``structure-sponge-loader`` addon to the pack manifest, using versions ``1.+``
+
+                2. Save your structure using `WorldEdit <https://worldedit.enginehub.org/en/latest/usage/clipboard/>`__.
+
+                3. Add the ``.schem`` file to your pack.
+
+                ``oak_tree.schem`` will be used for this guide.
+
+
+3. Create your feature config
+-----------------------------
+
+.. card::
+
+    Add the ``config-feature`` addon to the pack manifest, using versions ``1.+``.
+
+    This addon will allow us to create feature config files.
+
+    .. code-block:: yaml
+        :caption: pack.yml
+        :linenos:
+        :emphasize-lines: 7
+
+        id: YOUR_PACK_ID
+
+        version: 0.1.0
+
+        addons:
+          ...
+          config-feature: "1.+"
+
+    :ref:`Create a blank config file <create-config-file>` and open it your editor.
+
+    Set the :ref:`config type <config-types>` via the ``type``
+    parameter, and config ``id`` as shown below. ``oak_trees.yml`` will be used for this guide.
+
+    .. code-block:: yaml
+        :caption: oak_trees.yml
+        :linenos:
+
+        id: OAK_TREES
+        type: FEATURE
+
+4. Create the feature distributor
+---------------------------------
+
+.. card::
+
+    Distributors determine the x-axis and z-axis placement of a feature in the world.
+
+    Add the ``config-distributors`` addon to the pack manifest, using versions ``1.+``.
+
+    This addon will allow us to create distributors within feature config files.
```

Comment:
It would be more accurate to say the addon provides a set of distributors that can be used in a feature config